spec/checkPullRequestJobSpec.js

```diff
@@ -645,6 +660,10 @@ describe('Pull Request Job Spec', () => {
 
       jobs = checkPullRequestJobModule.getNewJobsFromFile(jobTestFile);
       expect(jobs.length).toBe(0);
+
+      jobs = checkPullRequestJobModule.getNewJobsFromFile(
+        modifiedExistingJobFileObjWithJobClassInPatch)
+      expect(jobs.length).toBe(0);
```

Comment:
Let us have a test where we have two jobs one just in patch and one actually added.

Reply:
Done.